wiki/Guides/Starting_a_Beatmap_Project/fr.md

```diff
@@ -31,6 +29,6 @@ Un exemple de titre incorrect
 
 - Exemples : [Anamanaguchi - Scott Pilgrim vs The World (The Game)](https://osu.ppy.sh/community/forums/topics/37908), [Daft Punk - Random Access Memories](https://osu.ppy.sh/community/forums/topics/132592)
 
-**Collab :** Créer un projet de mapping de groupe pour une certaine musique. Cela peut aller de la demande de storyboarding au mapping de certaines parties d'un marathon. [Référence](https://osu.ppy.sh/community/forums/topics/35841)
+**Collab :** Créer un [projet de mapping de groupe](/wiki/Beatmap/Beatmap_collaborations) pour une certaine musique. Cela peut aller de la demande de [storyboarding](/wiki/Storyboard) au mapping de certaines parties d'un [marathon](/wiki/Beatmap/Marathon). [Référence](https://osu.ppy.sh/community/forums/topics/35841)
 
-- Exemple : [Voca Nico Party Nonstop Mix - Marathon](https://osu.ppy.sh/community/forums/topics/188118)
+- Exemple : [Within Temptation ~ The Unforgiving](https://osu.ppy.sh/community/forums/topics/56629)
```

Comment:
do tell if the tilde is specific to french

```suggestion
- Exemple : [Within Temptation - The Unforgiving](https://osu.ppy.sh/beatmapsets/29157)
```

Reply:
@DoryanR is word before colon (`:`) need to have extra whitespace `Exemple :` or not `Exemple:` ?

Reply:
yes, if you check other french articles you can see that that is the case

Reply:
Alright I will also change `[Discographie]` to `[Discography]` so it can follow the forum rules. It would be confusing if there is french player open a thread with that prefix.